.travis.yml

```diff
@@ -1,25 +1,15 @@
-sudo: false
-language: python
-
-python:
-  - '2.7'
-
-install:
-  - pip install tox
-
-env:
-  - TOX_ENV=py27-test
-  - TOX_ENV=py33-test
-  - TOX_ENV=py34-test
-
-script:
-  - tox -e $TOX_ENV
-
-deploy:
-  provider: pypi
-  user: onefinestay
-  password:
-    secure: L838z+MtH79Px67frLMPo1LP/FULLDDlx6SOgZ/Npg+NBAs8D+Yu2OGCSLxgaHUU7FqK0Gqp4NNQxM8yLD1mGPbtgyd/DYlL8ugDHu45QEx+7YI2tAwc6XOKba1PXqYZi33/aCDUe5LYcVKfoNqXaXZxt25eWwI1VjnH5KJdEpI=
-  on:
-    tags: true
-    repo: onefinestay/nameko-sqlalchemy
+{
+  "sudo": false,
+  "language": "python",
+  "python": "2.7",
+  "install": [
+    "pip install tox"
+  ],
+  "env": "TOX_ENV=py27-test",
+  "script": [
+    "tox -e $TOX_ENV"
+  ],
+  "group": "stable",
+  "dist": "precise",
+  "os": "linux"
+}
```

Comment:
Woah JSON! Was this an intentional change?

Reply:
In other projects I've started using build stages to separate the different envs and deployment, e.g. https://github.com/Overseas-Student-Living/sqlalchemy-filters/blob/master/.travis.yml

The travis.yml we used to use won't work on the new Travis infrastructure because we relied on multiple pythons being available without explicitly selecting them... you used to be able to specify `python: 2.7` at the top level and tox could still find other pythons, but no longer.

Reply:
ignore the json please, I don't even know how it got committed .)

Reply:
I used the travis setup from filters